.github/workflows/link-validator.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         uses: coursier/setup-action@v1.3.3
 
       - name: Create the Pekko site
-        run: sbt -Dpekko.genjavadoc.enabled=true "Javaunidoc/doc; Compile/unidoc; docs/paradox"
+        run: sbt -Dpekko.genjavadoc.enabled=true -Dpekko.genlicensereport.enabled=false "Javaunidoc/doc; Compile/unidoc; docs/paradox"
```

Comment:
skip the `license-report` page generation in link-validator step.

Reply:
As this is a ci, can we keep it? or we add a dedicated task for `genlicensereport`